scripts/transactions/caps-to-foundation.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { Transaction } from '@mysten/sui/transactions';
+
+import { mainPackage } from '../config/constants';
+import { prepareMultisigTx } from '../utils/utils';
+
+const NETWORK = 'mainnet';
+const config = mainPackage.mainnet;
+
+// The new multisig address to transfer the caps to.
+const NEW_MULTISIG_ADDR = '0x9b388a6da9dd4f73e0b13abc6100f1141782ef105f6f5e9d986fb6e00f0b2591';
```

Comment:
confirming this is SUINS_FOUNDATION multisig 🫡

Reply:
key control on testnet: https://suiscan.xyz/testnet/tx/4K3dJJGDovdvTjzjcQGdgFuErQkm3c5VxVuyfp5Tyua5